src/ReactSelectize.ls

```diff
@@ -29,6 +29,7 @@ module.exports = create-class do
         # class-name :: String
         delimiters: []
         disabled: false
+        maxLength: 255
```

Comment:
You didn't understand me. If I were to use this library and happen to have values with more than 255 characters, upgrading to the version with your feature would break my code unless I specify the maxLength prop. It usually a better practice to make new features "opt-in", meaning not enforcing it unless wanted, and upgrading should not force you to make changes in your code.